pkg/proxy/iptables/proxier.go

```diff
@@ -257,6 +258,11 @@ func NewProxier(ipFamily v1.IPFamily,
 		klog.InfoS("nf_conntrack_tcp_be_liberal set, not installing DROP rules for INVALID packets")
 	}
 
+	if initOnly {
```

Comment:
so, we only have to plumb the initOnly just for `nf_conntrack_tcp_be_liberal` and `sysctlRouteLocalnet` in the iptables mode :/

Reply:
Actually there is, and shouldn't be, any difference between start with `--init-only` up until `Run()` is called. The rationale is here https://github.com/kubernetes/kubernetes/pull/120864#issuecomment-1734337301.